intl/msg_hash_ja.c

```diff
@@ -696,8 +696,8 @@ int menu_hash_get_help_jp_enum(enum msg_hash_enums msg, char *s, size_t len)
          break;
       case MENU_ENUM_LABEL_VALUE_MENU_ENUM_CONTROLS_PROLOG:
          snprintf(s, len,
-            "メニューを操作するのに \n"
-            "以下のゲームパッドまたはキーボードを使用できます: \n"
+            "メニューを操作するには \n"
+            "以下のゲームパッドまたはキーボードを使用します: \n"
```

Comment:
Given the current English translation, I think something like `以下の操作はゲームパッドまたはキーボードで使用できます` might work better. What do you think?

Reply:
The message you suggested doesn't include translation of "in order to control the menu", so I think my message is better.

Reply:
I meant to keep the above `メニューを操作するには` as well, so in that case it does include it.

Reply:
Is it your suggestion that the complete translation of this sentence is as follows?
"メニューを操作するには、以下の操作はゲームパッドまたはキーボードで使用できます"

If so, I feel it is a bit strange as a Japanese sentence, because the subject of the sentence is not "You" but "the following controls".
What do you think the following?
"メニューを操作するには、ゲームパッドまたはキーボードで以下の入力を使用できます"

Reply:
Ok, that works for me. Thanks